frog/imports/api/graphs.js

```diff
@@ -15,6 +20,36 @@ import {
 
 export const Graphs = new Mongo.Collection('graphs');
 
+export const createSessionFromActivity = (
+  activityType: string,
+  config: Object,
+  plane: number = 3
+): {
+  slug: string,
+  sessionId: string,
+  graphId: string,
+  activityId: string
+} | void => {
```

Comment:
NIT: Could move to its own type to improve readability and to allow callers to import the type